app/services/the_tvdb_service/row.rb

```diff
@@ -0,0 +1,87 @@
+class TheTvdbService
+  class Row
+    attr_reader :media_id, :media_type, :data, :episode_id, :tvdb_series_id
+
+    def initialize(media_id, data, tvdb_series_id)
+      @media_id = media_id
+      @data = data
+      @episode_id = data['id']
+      @tvdb_series_id = tvdb_series_id
+      @media_type = 'Anime'
+    end
+
+    def update_episode
+      return unless episode_number.present?
+
+      %i[
+        season_number relative_number synopsis
+        thumbnail airdate titles canonical_title
+      ].each { |k| public_send(k) }
+
+      # to get the imdbId we would have to call the /episodes/{episode_number} endpoint
+      # instead of the series/episdoes.
+      # TODO: implement getting imdbId once we can store it.
```

Comment:
(1) you should be more clear in the comment that the IMDB ID you're referring to is an *episode* ID
(2) you should specify what is meant by "once we can store it" (i.e. "once we can store mappings for episodes")